src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -1349,14 +1362,23 @@ private IEnumerable<string> BuildPackage()
                 Package.Keywords = KeywordList;
                 Package.License = License;
                 Package.SiteUrl = SiteUrl;
-                Package.RepositoryUrl = RepositoryUrl;
-                Package.HostDependencies = SelectedHosts;
+                Package.RepositoryUrl = RepositoryUrl;                
 
                 AppendPackageContents();
 
                 Package.Dependencies.Clear();
                 GetAllDependencies().ToList().ForEach(Package.Dependencies.Add);
 
+                Package.HostDependencies = Enumerable.Empty<string>();
+                Package.HostDependencies = SelectedHosts;
+                foreach (var py in GetPythonDependency()) 
+                {
+                    if (!Package.HostDependencies.Contains(py)) 
+                    {
+                        Package.HostDependencies = Package.HostDependencies.Concat(new[] { py });
```

Comment:
Would you remind me does this `HostDependencies` get saved into DB?

Reply:
yes